blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueRun.java

```diff
@@ -257,6 +265,45 @@
         public abstract Object getCause();
     }
 
+    @ExportedBean
+    public static final class BlueDownstreamRun {
+        private final String downstreamProject;
```

Comment:
Again this has nothing about folders I think?

Reply:
I based this on the way the upstream causactions work, do they fail with folders?

Reply:
Yes, you have to use upstreamUrl